### PR TITLE
Changed demo page and added Bootstrap to demo page

### PR DESCRIPTION
**Pull request by pandermatt**
Added Demo page and Demo alert

### CODE REVIEW

File: src/app/demo/custom-multi-upload/custom-multi-upload.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <p>{{i18n.t.upload.dropzone.subtitle}}</p></div>
+    <div *ngFor="let file of resultFiles">
+      <div class="col-xs-12">{{file.name}}
+        <a href="#" class="btn btn-danger btn-sm pull-right" (click)="removeFile(file, $event)" role="button">&times;</a>
+      </div>
+      <div class="col-xs-12">
+        <div class="progress">
+          <div class="progress-bar active"
+               [ngClass]="file.uploadProgressInPercent === 100 ? 'progress-bar-success' : 'progress-bar-striped'"
+               role="progressbar" aria-valuemin="0" aria-valuemax="100" [style.width]="file.uploadProgressInPercent + '%'">
```
Comment (pandermatt) [inline src/app/demo/custom-multi-upload/custom-multi-upload.component.html:19]: For Demo reason I inlined it

File: src/app/demo/demo.component.html [hunk 2]
```diff
@@ -1,32 +1,85 @@
...
+                       dimensions="400x400"
+                       quality="10"
+                       roundedCorners="100" background="10,10,255"
+                       watermark="https://www.renuo.ch/images/logo.png">
+      </ru-custom-image>
+    </div>
+  </div>
 
+  <hr>
+  <h1><code>ru-galler</code>
```
Comment (lukaselmer) [inline src/app/demo/demo.component.html]: typo

File: src/app/demo/scss/app.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
 body {
   font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+
+  margin: 0;
+
+  background-color: #4b93c2;
 }
```
Comment (lukaselmer) [inline src/app/demo/scss/app.scss:7]: > src/app/demo/scss/bootstrap/bootstrap.min.css

can we use a cdn instead?

**Review by lukaselmer — APPROVED** [review]
👍 LGTM

File: src/app/demo/scss/app.scss (hunk 3 above)
Comment (pandermatt) [reply src/app/demo/scss/app.scss:7]: with cdn also the non css one will be styled

**Pull request merged**